kansha/app/view.py

```diff
@@ -130,6 +130,8 @@ def render(self, h, comp, *args):
     h.head.javascript_url('js/kansha.js')
     h.head.javascript_url('js/autocomplete.js')
 
+    h.head.javascript_url('//cdn.ckeditor.com/4.5.3/standard/ckeditor.js')
```

Comment:
By using the CDN, we don't have to ship all the static stuff that comes with ckeditor (images, css, js) so we can keep the egg small.
From my point of view, this was the easiest way to add ckeditor to Kansha. I don't think that using a CDN can be considered a bad practice for an open source project.